tools/doxygen/Makefile

```diff
@@ -18,8 +18,8 @@ ifeq ($(HOST_OS),Windows)
   endif
 endif
 
-export docdirs = $(sort $(foreach dir,$(basedirs),${shell find $(contiking)/${dir} -type d}))
-export docsrc = $(docdirs) $(foreach dir,$(docdirs) .,${shell find $(dir) -type f -name "doxygen*.txt"})
+export docdirs = $(sort $(foreach dir,$(basedirs),${shell find $(contiking)/${dir} -type d -print0}))
```

Comment:
It seems like the argument -print0 change the behavior - it will not produce the full documentation anylonger.

Reply:
Ah alright. I'll take a look into that. :)